app/models/journal/wiki_content_journal.rb

```diff
@@ -30,3 +30,5 @@
 class Journal::WikiContentJournal < Journal::BaseJournal
   self.table_name = 'wiki_content_journals'
 end
+
+::WikiContentJournal = Journal::WikiContentJournal
```

Comment:
@ulferts as stated in e8c1d940e06c02740b289b952429ae5869598c1a, this is a hack for `Redmine::Acts::Journalized#journal_class` (a code monstrosity). I wanted to avoid having to rewrite (or even touch) journaling within this PR. If you can shed any light on a more elegant solution though, I'd be open to it.